python/salpytools/salpylib.py

```diff
@@ -283,8 +285,10 @@ def reply_to_transition(self, cmdId):
             LOGGER.info(
                 "WARNING: INVALID TRANSITION from {} --> {}".format(self.State.current_state,
                                                                     self.next_state))
-            self.State.send_logEvent('rejectedCommand', rejected_state=self.COMMAND)
-
+            # Send the ACK
+            self.mgr_ackCommand(cmdId, self.SAL__CMD_NOPERM, 0, "INVALID TRANSITION")
```

Comment:
I believe it's best practice to include the source and destination states in the message, just like the log message above.

Reply:
Good suggestion, I'll add that too.